cmd/main.go

```diff
@@ -33,6 +34,7 @@ const (
 	defBootstrapID                = ""
 	defBootstrapKey               = ""
 	defBootstrapRetries           = "5"
+	defBootstrapSkipTLS           = "false"
```

Comment:
Maybe better use `enable` than `skip`

Reply:
this how it is usually called in http library 
```
// Trust the augmented cert pool in our client
	config := &tls.Config{
		InsecureSkipVerify: skipTLS,
		RootCAs:            rootCAs,
	}
```